scripts/memTherm_core.py

```diff
@@ -405,7 +421,7 @@ def execute_core_hotspot(self, vdd_str):
      if (c_init_file_external!= "None") or (not first_run):
          c_hotspot_args += ' -init_file ' + c_init_file
 
-     #print hotspot_binary, hotspot_args
```

Comment:
we need not comment out the hotspot_binary printing. Adding the core hotspot printing in next line is good, but this should be also retained.